OpenRA.Game/GameRules/WeaponInfo.cs

```diff
@@ -39,9 +39,12 @@ public sealed class WeaponInfo
 		[Desc("The maximum range the weapon can fire.")]
 		public readonly WDist Range = WDist.Zero;
 
-		[Desc("The sound played when the weapon is fired.")]
+		[Desc("The sound played each time the weapon is fired.")]
 		public readonly string[] Report = null;
 
+		[Desc("Sound played only on first shot.")]
+		public readonly string[] FirstBurstReport = null;
```

Comment:
Please also add "in each salvo" or similar to the description, to make it extra clear.